src/sdrf_pipelines/parse_sdrf.py

```
@@ -258,28 +266,40 @@ def validate_sdrf(
+        contents = []
+        for template in templates:
+            try:
+                if hasattr(registry, "raw_schema_data") and template in registry.raw_schema_data:
+                    contents.append(yaml.dump(registry.raw_schema_data[template], sort_keys=True))
+                else:
+                    schema_dir = os.path.join(os.path.dirname(__file__), "sdrf", "schemas")
+                    template_file = os.path.join(schema_dir, f"{template}.yaml")
+                    if os.path.exists(template_file):
+                        with open(template_file, "r", encoding="utf-8") as f:
+                            contents.append(f.read())
+            except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)